File: src/1347-Minimum-Number-of-Steps-to-Make-Two-Strings-Anagram/1347.js
var minSteps = function(s, t) {
    let cnt = new Array(26), n = s.length, res = 0;
    cnt.fill(0);
    
    for (let i = 0; i < n; i++) {
        ++cnt[s[i].charCodeAt() - 97];
        --cnt[t[i].charCodeAt() - 97];
    }
    
    for (let i = 0; i < 26; i++) {
        if (cnt[i] > 0) res += cnt[i];
    }
    return res;
};

File: src/1347-Minimum-Number-of-Steps-to-Make-Two-Strings-Anagram/1347.go
func minSteps(s string, t string) int {
    cnt := make([]int, 26)
    n, res := len(s), 0
    
    for i := 0; i < n; i++ {
        cnt[s[i] - 'a']++
        cnt[t[i] - 'a']--
    }
    
    for i := 0; i < 26; i++ {
        if cnt[i] > 0 {
            res += cnt[i]
        }
    }
    return res
}